apps/src/openni_mls_smoothing.cpp

```diff
@@ -220,26 +216,23 @@ main (int argc, char ** argv)
   double sqr_gauss_param = default_sqr_gauss_param;
   bool sqr_gauss_param_set = true;
   int polynomial_order = default_polynomial_order;
-  bool use_polynomial_fit = default_use_polynomial_fit;
 
   pcl::console::parse_argument (argc, argv, "-search_radius", search_radius);
   if (pcl::console::parse_argument (argc, argv, "-sqr_gauss_param", sqr_gauss_param) == -1)
     sqr_gauss_param_set = false;
-  if (pcl::console::parse_argument (argc, argv, "-polynomial_order", polynomial_order) == -1 )
-    use_polynomial_fit = true;
-  pcl::console::parse_argument (argc, argv, "-use_polynomial_fit", use_polynomial_fit);
+  pcl::console::parse_argument (argc, argv, "-polynomial_order", polynomial_order)
```

Comment:
CI just failed. Sintax error here. Needs the semicolon at the end.

Reply:
And how on Earth did it succeed to build locally? Mystery.